client/app/scripts/liveblog-syndication/actions/incoming-syndication.js

```diff
@@ -27,6 +27,23 @@ liveblogSyndication
                             });
                         });
                     });
+                },
+                publish: function(post) {
+                    postsService.savePost(post.blog, post, undefined, {post_status: 'open'})
+                        .then(function(post) {
+                            Dispatcher.dispatch({
+                                type: 'ON_YANKED_POST',
```

Comment:
Both actions triggers the same reducer, which removes an entry from a the posts list. Hence the "yank"